app/domain/authentication/authn_jwt/orchestrate_jwt_authentication.rb

```diff
@@ -6,25 +6,30 @@ module AuthnJwt
 
     OrchestrateJwtAuthentication ||= CommandClass.new(
       dependencies: {
-        token_factory: TokenFactory.new
+        jwt_configuration_factory: JwtConfigurationFactory.new,
+        jwt_authenticator: Authentication::AuthnJwt::Authenticate.new
       },
       inputs: %i[authenticator_input]
     ) do
       extend(Forwardable)
       def_delegators(
-        :@authenticator_input, :account, :username
+        :@authenticator_input
       )
```

Comment:
looks like you're not using the delegators so you can remove them. unless you want them as a placeholder for now which is fine

Reply:
Removed